IP_POLICY_GUIDANCE.md

```diff
@@ -0,0 +1,90 @@
+# OpenJS Foundation IP Policy Guidance
+
+The purpose of this guide is to make it straightforward for projects
+to follow the [OpenJS Foundation IP Policy][IP Policy].
+
+Of course, the source of truth remains the [IP Policy][].
+
+If in doubt about anything related to IP after reading this document,
+first read the [IP Policy][], and then [ask for help](#getting-help).
+
+## Basic requirements
+
+There are four key requirements for all OpenJS Foundation projects:
+
+1. [Licensing](#1-licensing)
+2. [Copyright notices](#2-copyright-notices)
+3. Displaying the [standard website footer](#3-standard-website-footer) on the project's website
+4. Choose between the [DCO][] or a [Contributor License Agreement (CLA)](#4-contributor-license-agreement-cla)
+
+### 1. Licensing
+
+Code may be licensed under:
+  * [Apache, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+  * [MIT](https://opensource.org/licenses/MIT)
+  * [2-Clause BSD](https://opensource.org/licenses/BSD-2-Clause)
+  * [3-Clause BSD](https://opensource.org/licenses/BSD-3-Clause)
+
+Documentation may be licensed under:
+  * [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)
+  * [MIT](https://opensource.org/licenses/MIT)
+
+If you need to use a license which isn't in the [IP Policy],
+you will need to obtain permission from the Board to do so. Please [open an issue in the CPC repo](https://github.com/openjs-foundation/cross-project-council/issues), and assign it to brianwarner.
+
+### 2. Copyright notices
+
+The recommendation is to use a general copyright statement
+of the following form (where XYZ is the project's name):
+
+- **Copyright The XYZ Authors.**
+- **Copyright The XYZ Contributors.**
+- **Copyright Contributors to the XYZ project.**
+
+By using this format, the project avoids having to deal with
+names of copyright holders, years or ranges of years,
+and variations on the (c) symbol.
+
+_Please note that you **must not** change or remove existing copyright lines
+unless you put them there **and** have the right to do so.
+If there are existing copyright lines
+add the recommended copyright statement below the existing copyright lines.
+You may however update copyright lines that say
+"JS Foundation" or "Node.js Foundation" to "OpenJS Foundation"
+as this simply reflects the merged status of the organizations._
+
+### 3. Standard website footer
+
+All project websites need to have a [standard website footer][].
+
+### 4. Contributor License Agreement (CLA)
+
+By default the OpenJS Foundation uses the [DCO][], which means contributors
+submit their contributions under the project's designated open source license.
+Every commit must contain a Signed-off-by line in the commit message (e.g.,
+`Signed-off-by: Authorname <author@email.com>`), indicating the author has the
+right to make the contribution under the open source license.
```

Comment:
This feels like an implementation detail of the DCO that begs more questions:

- What happens when there are multiple authors?
- What happens when commits are squashed?
- Can I sign off for someone else when I merge a commit?
- What if I'm batch-merging commits made with GitHub's "insert a suggestion" feature?
- Is a Sign-off-by in the body of the pull request good enough?
- Etc.

My suggestion would be to have a separate documents with implementation details and best practices for implementing the DCO and CLA and dealing with them on a day to day basis.

Reply:
In general those others are definitely implementation details and would be great in a breakout, but I strongly recommend keeping this as it explains what the DCO is and how it differs from the next option. I suspect many don't make the association between a signed-off-by line and the DCO (much as many don't realize that there's only one DCO).

Reply:
OK, so I tried to include a small description of what needs to happen from a maintainer's perspective to implement the DCO. I have more questions: Does the project need to mention the DCO anywhere? What's language should it use? Should it set up infra? Etc.

Reply:
That's a good point, I think it would be worth me putting together some guidelines on this. I'll take this one on.